components/organizer/OrganizerScreen.tsx

```diff
@@ -118,16 +114,56 @@ function OrganizerScreen(props: SurveyProps) {
                 })
               )}
         </List.Accordion>
+        <View style={styles.separator} />
+      </ScrollView>
 
-        <Button
-          style={styles.email}
-          mode="contained"
+      <View
+        style={{
+          display: "flex",
+          flexDirection: "row",
+          alignItems: "center",
+          backgroundColor: theme.colors.surface,
+          justifyContent: "space-between",
+          width: "70%",
+          borderRadius: 50,
+          position: "absolute",
+          padding: 5,
+          bottom: 30,
+          alignSelf: "center",
+          shadowColor: "grey",
+          shadowOffset: {
+            width: 0,
+            height: 5,
+          },
+          shadowOpacity: 0.1,
+          shadowRadius: 2,
+        }}
+      >
+        <FAB
+          style={{ backgroundColor: theme.colors.exit }}
+          icon="close"
+          onPress={() => {
+            props.updateAuthentication({
+              isOrganizer: false,
+              surveyId: "",
+              responseId: null,
+            });
+            props.updateAnswer([]);
+          }}
+        />
```

Comment:
looks like this could be in the `FinishButton` maybe? though this button might look different in different screens so maybe not